src/main/java/com/neuvector/Scanner.java

```diff
@@ -128,7 +128,7 @@ public static ScanRepoReportData scanLocalImage(Image image, NVScanner nvScanner
                 .withUserAndGroup(getDockerUserGroupCmdArg(getScanReportPath(nvScanner.getNvMountPath())))
                 .withName(generateScannerName())
                 .withVolume(Scanner.SOCKET_MAPPING)
-                .withVolume(getMountPath(nvScanner))
+                .withVolume(appendBindMountSharedSuffixIfRequired(nvScanner.isBindMountShared(), getMountPath(nvScanner)))
```

Comment:
If I am reading the code correctly, the `nvScanner` object should have `log`. Please log the final `cmdArgs` before the `runScan` call. Same for `scanLocaIImage`.

Reply:
Commit: [95ff98f](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/95ff98fa748659b82ad5527cf1b202fa6f2df662)